bamboo/set-bamboo-env-variables.sh

```diff
@@ -101,27 +102,28 @@ fi
 ## Set integration stack name if it's not been overridden *or* set by SIT
 if [[ -z $DEPLOYMENT ]]; then
   DEPLOYMENT=$(node ./bamboo/select-stack.js)
+
+  if [[ $KES_DEPLOYMENT != true ]]; then
+    echo "Using terraform stack name $DEPLOYMENT-tf"
+    DEPLOYMENT=$DEPLOYMENT-tf
+  fi
+
   echo deployment "$DEPLOYMENT"
   if [[ $DEPLOYMENT == none ]]; then
     echo "Unable to determine integration stack" >&2
     exit 1
   fi
-  if [[ $COMMIT_MESSAGE =~ deploy-terraform || $BRANCH =~ terraform ]]; then
-    echo "Detected terraform deployment branch or commit"
-    echo deployment "$DEPLOYMENT-tf"
-    DEPLOYMENT="$DEPLOYMENT-tf"
-  fi
   echo export DEPLOYMENT=$DEPLOYMENT >> .bamboo_env_vars
 fi
 
-if [[ $DEPLOYMENT =~ '-tf' ]]; then
+if [[ $KES_DEPLOYMENT != true ]]; then
   echo "Using NGAPShNonProd credentials"
   export AWS_ACCESS_KEY_ID=$bamboo_SECRET_NONPROD_AWS_ACCESS_KEY_ID
   export AWS_SECRET_ACCESS_KEY=$bamboo_SECRET_NONPROD_AWS_SECRET_ACCESS_KEY
 fi
 
 export PR_BRANCH=master
-if [[  $DEPLOYMENT =~ '-tf' ]]; then
+if [[  $KES_DEPLOYMENT != true ]]; then
   echo "Setting GIT_PR target branch to 'terraform'"
   export PR_BRANCH=terraform
 fi
```

Comment:
This needs to go as soon as we put the PR in for master

Reply:
Or not if we're continuing to support Kes deployments for a while?